front/src/reducers/osrdconf/stdcmConf/stdcmConfReducers.spec.ts

```diff
@@ -269,16 +257,11 @@ describe('stdcmConfReducers', () => {
         },
       };
 
-      store.dispatch(updateLastStdcmResult(newSimulation));
+      store.dispatch(addStdcmSimulation(newSimulation));
       const state = store.getState()[stdcmConfSlice.name];
-      expect(state.simulations.length).toEqual(1);
-      expect(state.simulations.at(0)).toEqual(newSimulation);
-      expect(state.selectedSimulationIndex).toEqual(0);
-      expect(state.totalLength).toEqual(newSimulation.inputs.consist!.totalLength);
-      expect(state.totalMass).toEqual(newSimulation.inputs.consist!.totalMass);
-      expect(state.maxSpeed).toEqual(newSimulation.inputs.consist!.maxSpeed);
-      expect(state.speedLimitByTag).toEqual(newSimulation.inputs.consist!.speedLimitByTag);
-      expect(state.stdcmPathSteps).toEqual(newSimulation.inputs.pathSteps);
+      expect(state.simulations.length).toEqual(2);
+      expect(state.simulations[0].inputs).toEqual(simulationInputs);
+      expect(state.simulations[0].index).toEqual(0);
```

Comment:
Hum you should be testing the 2nd element of the array no ?
```suggestion
      expect(state.simulations[1].inputs).toEqual(simulationInputs);
      expect(state.simulations[1].index).toEqual(0);
```